charts/kubewarden-controller/templates/webhooks.yaml

```diff
@@ -1,9 +1,48 @@
+# generate certificates
+{{ $dnsName :=  printf "%s-webhook-service.%s.svc" (include "kubewarden-controller.fullname" .) .Release.Namespace }}
+{{ $ca := genCA "kubewarden-controller-ca" 365 }}
+{{ $cert := genSignedCert $dnsName nil ( list $dnsName ) 3650 $ca }}
+{{ $caCert := ($ca.Cert | b64enc) }}
+{{ $caPrivatKey := ($ca.Key | b64enc) }}
+{{ $serverCert := ($cert.Cert | b64enc) }}
+{{ $serverPrivateKey := ($cert.Key | b64enc) }}
+# check if the secrets already exist and if so, use the existing values
+{{ $caSecret := (lookup "v1" "Secret" .Release.Namespace "kubewarden-ca") }}
+{{ if $caSecret }}
+{{ $caCert = (index $ca.data "ca.crt") }}
+{{ $caPrivateKey = (index $ca.data "ca.key") }}
+{{ end }}
+{{ $serverCertSecret := (lookup "v1" "Secret" .Release.Namespace "kubewarden-webhook-server-cert") }}
+{{ if $serverCertSecret }}
+{{ $serverCert = (index $serverCertSecret.data "tls.crt") }}
+{{ $serverPrivateKey = (index $serverCertSecret.data "tls.key") }}
+{{ end }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kubewarden-ca
+  namespace: {{ .Release.Namespace }}
+labels:
+  {{- include "kubewarden-controller.labels" . | nindent 4 }}
+data:
+  ca.crt: {{ $caCert }}
+  ca.key: {{ $caPrivatKey }}
```

Comment:
I think we have to lookup also for the oldCA certificate and make sure helm doesn't wipe it out accidentally